test/vaadin-radio-group.html

```diff
@@ -497,6 +497,7 @@
       });
 
       describe('change event', () => {
+
```

Comment:
not a blocker, but please remove this newline, it's not needed.

Reply:
removed it